### DISCUSSION TITLE
figures is not listed as one of the dependencies

### DISCUSSION THREAD

**Discussion by datsfilipe** [eduardconstantin/storybook-genie#7] [General]
No problems when running with `npx` but at least for local development it probably should be included, I think.

**Comment by eduardconstantin** [comment]
oh, you mean this one?
[https://www.npmjs.com/package/figures](https://www.npmjs.com/package/figures)

**Comment by datsfilipe** [comment]
> oh, you mean this one?
> [https://www.npmjs.com/package/figures](https://www.npmjs.com/package/figures)

Yes

**Comment by eduardconstantin** [comment]
Good catch! I forgot do add it. Thank you!
